### PR TITLE
i18n(it): Added 'MPA vs SPA' page

### PR DESCRIPTION
**Pull request by csarnataro**
#### What kind of changes does this PR include?

- Translated content

#### Description

- Closes # N/A 
- Added the 'MPA vs SPA' page for Italian docs

### CODE REVIEW

**Comment by vclemenzi** [comment]
LGTM!

File: src/content/docs/it/concepts/mpa-vs-spa.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+#### Gestione dello stato lato server (MPA) e lato client (SPA) a confronto
+
+Le SPA, per via della loro architettura, sono migliori per siti che hanno una gestione complessa e multi-pagina dello stato (per esempio: Gmail). Questo perché una SPA esegue l'intero sito web come una singola applicazione JavaScript, consentendo all'applicazione di mantenere lo stato e la memoria anche navigando tra pagine diverse. Esperienze altamente interattive e guidate dai dati, come caselle di posta elettronica o pannelli di amministrazione, sono più adatte ad essere implementate con SPA, in quanto sono di per sé più simili ad "app".
+
+
+## Le MPA sono meglio delle SPA?
+
+Nel confronto tra MPA e SPA, non c'è "meglio" o "peggio". Tutto dipende da quello che si vuole ottenere e relativi compromessi.
+
+** Astro predilige le performance e la semplicità delle MPA perché sono più adatte al caso d'uso di siti web incentrati sui contenuti. ** Altri siti più interattivi, con una complessa gestione dello stato interno, possono trarre più benefici da un'architettura SPA più simile alle "app", anche a costo di performance inferiori durante il caricamento iniziale.
```
Comment (Yan-Thomas) [inline src/content/docs/it/concepts/mpa-vs-spa.mdx:52]:
```suggestion
**Astro predilige le performance e la semplicità delle MPA perché sono più adatte al caso d'uso di siti web incentrati sui contenuti.** Altri siti più interattivi, con una complessa gestione dello stato interno, possono trarre più benefici da un'architettura SPA più simile alle "app", anche a costo di performance inferiori durante il caricamento iniziale.
```

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for this peeps! 🚀
I went ahead and fixed a bold formatting issue, hope you don't mind!

**Comment by milovangudelj** [comment]
Line 4 (`i18nReady: false`) should be removed from translated documents. It's only meant to be used in the original english version.

**Pull request merged**